src/rgw/rgw_sal_dbstore.cc

```diff
@@ -1847,7 +1847,7 @@ namespace rgw::sal {
 
   int DBStore::get_config_key_val(string name, bufferlist *bl)
   {
-    return 0;
+    return -1;
```

Comment:
Done

Reply:
Yes please....
Use error codes from errno.h as much as possible.
At the moment I'm trying to figure out why the test for this is giving me a hard time.
Some of the suff is because things like `ECANCEL` in the tests are hardcoded as a value.

I'm old, but my coding 101 told me to avoid litterals as much as possible.
And in the case of comparing errors there are better alternatives